janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -562,17 +567,12 @@ public void clearStorage() throws BackendException {
 
     @Override
     public List<KeyRange> getLocalKeyPartition() throws BackendException {
-
         List<KeyRange> result = new LinkedList<KeyRange>();
-
         HTable table = null;
         try {
-            ensureTableExists(tableName, getCfNameForStoreName(GraphDatabaseConfiguration.SYSTEM_PROPERTIES_STORE_NAME), 0);
-
-            table = new HTable(hconf, tableName);
```

Comment:
You deleted the assignment to `table` and it doesn't look like `table` is used anywhere. Might as well delete the `IOUtils.closeQuietly(table)` and the declaration of `table` as well?